app/views/templates/_form.html.erb

```diff
@@ -84,6 +83,74 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="help">
+      <div>
+        <%= _('Provisioning templates are generated using Embedded Ruby (ERB) language.') %>
+        <%= _('The following table and example snippet shows global methods, variables and class methods and how to use them.') %>
+        <%= _('These are guaranteed to work via the safe mode rendering, to ensure a template can do nothing harmful.') %>
+        <% unless Setting[:safemode_render] -%>
+          <%= _('To enable safe mode, navigate to Settings page and flip Safemode rendering option.') %>
+        <% end -%>
+        <code class="transparent"><pre>
```

Comment:
Is `<code>` superfluous here? Pre should keep formatting, and also doesn't appear to need the CSS override.